transport/TransportImpl.cpp

```diff
@@ -1065,8 +1065,10 @@ void TransportImpl::internalIceReceived(Endpoint& endpoint,
             return;
         }
 
-        if (_rtpIceSession->getState() == ice::IceSession::State::CONNECTING)
+        //logger::debug("ice received, ICE state %u", _loggableId.c_str(), _rtpIceSession->getState());
+        if (_rtpIceSession->getState() != ice::IceSession::State::CONNECTED)
```

Comment:
Not usre about this yet. Something goes wron un unreg so there is a ref count 1 on transportimpl.